main.go

```diff
@@ -579,6 +650,8 @@ func parseFlags() (config, error) {
 			" Note that TLS 1.3 ciphersuites are not configurable.")
 	flag.DurationVar(&cfg.tls.reloadInterval, "tls.reload-interval", time.Minute,
 		"The interval at which to watch for TLS certificate changes.")
+	flag.StringVar(&cfg.rateLimiterAddress, "middleware.rate-limiter.grpc-address", "",
+		"The gRPC Server Address against which to run rate limit checks. If it is not specified local non-shared rate limiting will be used.")
```

Comment:
```suggestion
		"The gRPC Server Address against which to run rate limit checks. when rate limits are specified for any tenant. If it is not specified local non-shared rate limiting will be used.")
```

Reply:
Wonder.. shouldn't this be option for config above? 

```
RateLimits []*struct {
			Endpoint string   `json:"endpoint"`
			Limit    int      `json:"limit"`
			Window   duration `json:"window"`
		} `json:"rateLimits"`
```

Reply:
What exactly do you mean? Do you want to specify rate limiter endpoint per tenant?